packages/doctype-caip10-link/src/caip10-link-doctype.ts

```diff
@@ -59,7 +67,7 @@ export class Caip10LinkDoctype extends Doctype {
      * create a verifiable link from the CAIP10 address to the DID.
      * @param did - The DID being linked to the CAIP10 address that this Caip10Link object represents.
      *   If the 'did' provided is an instance of the DID type, the DID must already be authenticated
-     *   so that the did string it represents is available.
+     *   so that the did string it represents is avapilable.
```

Comment:
Wut? 😆

Reply:
oops!

Reply:
In my defense I hadn't actually sent this for review yet, I was still gonna do a proofreading pass.  But you're just too quick for me!  :laughing: